docs/rfcs/0004-suggested-excluded-apps-tech-spec.md

```diff
@@ -0,0 +1,141 @@
+Suggesting Apps for Split Tunneling
+===================================
+
+Epic: [VPN-4412: Suggest Apps to Exclude from VPN Protection](https://mozilla-hub.atlassian.net/browse/VPN-4412)
+
+Context / Problem Statement
+---------------------------
+
+We're looking to create tools that prompt the user in ways that help them stay more secure. Split tunneling (called 'App Exclusion' to users) is a feature that can reduce the need to turn off the VPN entirely by excluding traffic in specific apps from the VPN tunnel, but many users don't know about it. The current feature is in settings, and provides an alphabetical checklist of installed apps:
+
+<img src="./images/0004-01.png" height=500>
+
+This epic is for a new feature to proactively let users know when they have apps that could benefit from the split tunneling feature.
```

Comment:
Could we add a slightly more detailed overview of how this feature works? I think this can mostly be inferred from the tasks but it would be helpful to have it summarized somewhere.